environments/budgetkey/values.yaml

```diff
@@ -85,7 +85,7 @@ pipelines:
   # see the pipelines configmap for the required keys
   secretName: pipelines
   resources: >
-    {"requests": {"cpu": "800m", "memory": "2Gi"}, "limits": {"cpu": "1500m", "memory": "5Gi"}}
+    {"requests": {"cpu": "200m", "memory": "2Gi"}, "limits": {"memory": "3Gi"}}
```

Comment:
What does it mean if there's no CPU limit?

Reply:
extra available cpu time above the requested cpu is distributed among the pods based on the requests cpu - e.g. a pod that requests more cpu will get more extra cpu (when available / needed)